### ISSUE TITLE
Tests broken for v2.0.0

### ISSUE THREAD

**Issue by abulimov** [Orolia2s/disciplining-minipod#4]
It looks like tests were not updated, and currently don't work on v2.0.0.

They don't even compile as of now (lots of compilation problems), and running them after removing compilation flags like `-Wall -Werror` from the Makefile reveals that compiled tests fail on first assertion:

```
TEST utils program
Testing simple linear reg
a = 1.576194
b = 0.000406
test-utils: test-utils.c:45: main: Assertion `test_result.a == 20.0' failed.
```

**Comment by CharlesParent** [comment]
Hi @abulimov !
Normally tests have been fixed in version 2.1.0 ! Let me know if you have any trouble !

**Comment by abulimov** [comment]
Thanks, confirmed tests are passing now.
